src/index.ts

```diff
@@ -151,6 +152,10 @@ function preactPlugin({
 						},
 					},
 				},
+				esbuild: {
+					jsx: "automatic",
+					jsxImportSource: jsxImportSource ?? "preact",
```

Comment:
IIrc there is a config option to enable the jsx dev transform in esbuild too.

Reply:
Yeah could do, but that'd really only come up if the user has explicitly disabled devtools, which I'm (anecdotally) guessing is uncommon?

Reply:
Ah yeah, it's a bit unintuitive, but it's a Babel or ESBuild situation, not both.

Will add a comment to point that out.